fe/src/main/java/org/apache/doris/catalog/TabletInvertedIndex.java

```diff
@@ -298,57 +262,28 @@ public Long getTabletIdByReplica(long replicaId) {
         }
     }
 
-    public long getPartitionId(long tabletId) {
-        readLock();
-        try {
-            if (tabletMetaMap.containsKey(tabletId)) {
-                return tabletMetaMap.get(tabletId).getPartitionId();
-            }
-            return NOT_EXIST_VALUE;
-        } finally {
-            readUnlock();
-        }
-    }
-
-    public long getIndexId(long tabletId) {
+    public TabletMeta getTabletMeta(long tabletId) {
         readLock();
         try {
-            if (tabletMetaMap.containsKey(tabletId)) {
-                return tabletMetaMap.get(tabletId).getIndexId();
-            }
-            return NOT_EXIST_VALUE;
+            return tabletMetaMap.get(tabletId);
         } finally {
             readUnlock();
         }
     }
 
-    public int getEffectiveSchemaHash(long tabletId) {
-        // always get old schema hash(as effective one)
+    public List<TabletMeta> getTabletMetaList(List<Long> tabletIdList) {
```

Comment:
It is very error-prone to result a list will null in it.
How about using `getOrDefault()` method, and return an `NonExistTabletMeta` is not found?
And `NonExistTabletMeta` can be a `final static` member with all id as `NON_EXIST_VALUE` in it.

In this way, we don't have to check null each time we visit the returned list.

Reply:
ok